Documentation/Open Location Code API Specification.md

```diff
@@ -0,0 +1,78 @@
+# Open Location Code API Reference Specification
+
+## Version history
+
+All substantial changes changes to this document are listed here.
+
+* Version 1.1.0 / 2021-06-04 / [William Entriken](https://github.com/fulldecent/)
+  * Update to match nomenclature in [Plus Codes specification](./Plus%20Codes%20Specification.md)
+  * Separate REQUIRED and OPTIONAL public methods
+  * Specify the number of significant digits that a short code Plus Code may omit
+
+- Version 1.0.0 / 2014-10-27 / [Doug Rinckes](https://github.com/drinckes), Google / Philipp Bunge, Google
+  - Initial public release
+
+## REQUIRED public methods
+
+An implementation of the Open Location Code API SHALL implement these REQUIRED methods:
+
+| Method name          | Input(s)                        | Output(s)                                                    |
+| -------------------- | ------------------------------- | ------------------------------------------------------------ |
+| `isFull`             | Plus Code string                | True if this is a long code Plus Code, false otherwise       |
```

Comment:
Already added as a PR to yours, but now in review form: "long code Plus Code" is not a defined term and should be replaced with "full code Plus Code".

Aside: This is a mouthful - can we perhaps define just "full code" and "short code" early on and from then on omit always adding "Plus Code"?

Reply:
I agreed // I had been overzealous in fully qualifying all references to "short code" and "full code". Fixed at a896494bc7fd0b640c71d04b0ac4bb7c1263f7a5